API/Startup.cs

```diff
@@ -49,7 +53,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
 
             app.UseRouting();
 
-            app.UseCors(X => X.AllowAnyHeader().AllowAnyMethod().WithOrigins("https://localhost:4200"));
+            app.UseCors(X => X.AllowAnyHeader().AllowAnyMethod().WithOrigins(_config["Origin"]));
```

Comment:
Origins should be an array, and better to configure the settings to make it as an array